cypress/e2e/group/showGroupsAndClassesInTable.feature

```diff
@@ -0,0 +1,83 @@
+@release
+Feature: Group - To show groups and classes in one table with respective functionality
+
+  As a teacher I want to see all groups and classes belonging to my school.
+
+  @stable_test
+  Scenario: As a pre-condition teacher adds a class to school
+    Given I am logged in as a 'teacher1_nbc' at 'nbc'
+    When I go to administration page
+    And I go to class administration
```

Comment:
we don use 'And' in feature now...so it should be like:
Given-  is for the pre conditions like login
When- is for the intermediate actions
Then - is for the expected outcome/acceptance criteria